package.json

```diff
@@ -17,7 +17,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/RevenueCat/purchases-capacitor.git"
+    "url": "https://github.com/RevenueCat/purchases-capacitor.git"
```

Comment:
Oh makes sense. Just a question, but does it need to finish with `.git`? (it's fine if it does, just confirming)

Reply:
Tbh, this was "minimum viable change" so that `pod trunk push` works. This url It's redirected to the same url without the `.git`, but I don't know if this is actually required for CocoaPods 🤔